### ISSUE TITLE
Standard list of activity tags

### ISSUE THREAD

**Issue by quozl** [sugarlabs/sugar-toolkit-gtk3#399]
Recent development as part of GSoC 2018 has added *tags* to many of the core Fructose activities, with several side-effects showing up during integration;

* the List View  (<kbd>F2</kbd> <kbd>ctrl</kbd>+<kbd>2</kbd>) shows the tags in a comma separated list under the activity name,

* the List View search feature finds activities with tags,

* the tags are not translated into the configured language,

* the tags are somewhat inconsistently applied.

*tags* is briefly defined in https://github.com/sugarlabs/sugar-toolkit-gtk3/blob/master/src/sugar3/bundle/__init__.py#L72

We need;

* [ ] a standard list of tags,
* [ ] translations of the tags.

CC: @vipulgupta2048 

Screenshot of List View after searching for "documen";

![screenshot_ubb_2018-08-09_15 07 06](https://user-images.githubusercontent.com/2936481/43879203-222182d2-9be6-11e8-9ac4-d05aebe9e802.png)

**Comment by vipulgupta2048** [comment]
> a standard list of tags

Like @quozl mentioned and after going over a lot of activities for my project in GSoC 2018. The `tags` parameter was introduced to maintain uniformity in terms of the classification of activities. The list of tags below can be considered as standard if the community agrees on it. 

Programming, Robotics, Internet, Science, Maths, Language, Geography, Documents, Music, Media, Art, Teacher, or System.

Apart from the tags listed above, I have also proposed the use of `Tools` which can be classified as activities that are essential and increases productivity. I might not have noticed `Utilities` not being the list hence the tag being in write activity,

SO, if we could have a vote between `Utilities` or `Tools`, then we could add the same in #400

**Comment by quozl** [comment]
@vipulgupta2048, actually the `tags` metadata feature was already there, it's just that your project began to use it for aslo-v3 and so you began populating it.  That's why it is starting to show up in List View.

`Tools` is as ambiguous as `Activity`.  I don't think `Tools` has any meaning.

`Utilities` would more commonly mean programs that act on system or storage components; like the Journal, Frame, Neighbourhood View, or the hidden Log and Terminal activities.

That productivity is increased is a value judgement disconnected from child learning; the Maze game will increase productivity by improving visual perception; the Implode game will increase productivity by improving problem solving ability, etc.

@walterbender, is there a software classification system used in pedagogy?

**Comment by tony37** [comment]
I can see no benefit in cluttering the List view with these tags. The 
activity has been installed or it wouldn't be visible in the List View. 
The user's only options are to delete the activity or to make it visible 
in the Home view.

Sugar is deviating grossly from its guidelines. This change adds to the 
translation burden. The VIews were designed to be usable by icons not 
requiring words.
Now young learners whose native language is not English will need to 
cope with more words which may be important to their use of Sugar but 
can be safely ignored. Teachers will need to be trained that these 
categories are not relevant to their students - who will inevitably 
raise thier hands to ask.

Tony

**Comment by quozl** [comment]
The presentation of tags was [added by Tomeu in 2009](https://github.com/sugarlabs/sugar/commit/014a004eb2a708bd368f494e98a6c3705495cf76), so it not true to say this is a gross deviation.  Also it only appears in List View, so hand raising will be a good learning opportunity.

Adding tags to activities as has just happened does however add a translation burden, and that's my biggest concern.  If this can't be resolved, I'll pull them out.  @leonardcj, as translations manager, what's your opinion?

@vipulgupta2048, translation is handled by a team who use https://translate.sugarlabs.org/ ... use of Google Translate is often unreliable, as there is insufficient context for the algorithm to pick a word from several.

**Comment by tony37** [comment]
I don't remember the details - but at that time the intent was to enable 
a search by tags (primarily the Journal). I don't believe this was in 
connection with a 'standard list' but more with tags supplied by the 
user in the language of their choice.

Tony

**Comment by quozl** [comment]
No, the Journal metadata tags are entirely different.  Here in this issue we are dealing with activity tags, which enhance the search of the activity List View.
